webpack.dev.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const merge = require('webpack-merge');
+const exec = require('child_process').exec;
```

Comment:
I think you had this from readthedocs, right? Maybe a short comment giving attribution would be good then

Reply:
My basis where I started from is indeed from readthedocs, the webpack.common file did change so much only basic boilerplate remains.

Giving attribution is a good idea. Maybe we should do this in the theme dev documentation?